### PR TITLE
Endpoints working with the database

### PR DESCRIPTION
**Pull request by alexeindev**
All the endpoints are working as the user story requires. We are talking with the modules team and they already made a class that will help us with the repeated code that it's a new step to refactor our code.

### CODE REVIEW

**Review by daniel33gomez — APPROVED** [review]

File: src/main/server.js [hunk 1]
```diff
@@ -7,25 +7,28 @@ import { Scenario } from './modules/Scenario';
 import { Execution } from './modules/Execution';
 import { Output } from './modules/Output';
 
+const COLECTION_NAME = 'MSM-Scenario';
```
Comment (daniel33gomez) [inline src/main/server.js:10]: My lonely consideration is that COLECTION_NAME shoud be COLLECTION_NAME. With double L

**Review by JeanPeare — APPROVED** [review]

File: src/main/server.js [hunk 2]
```diff
@@ -7,25 +7,28 @@ import { Scenario } from './modules/Scenario';
...
+        extended: true,
+    })
+);
 
 // DB configuration
 // const serviceAccount = process.env.GOOGLE_APPLICATION_CREDENTIALS;
 admin.initializeApp({
     credential: admin.credential.cert(serviceAccount),
-    databaseURL: 'https://metal-slug-maker-default-rtdb.firebaseio.com'
+    databaseURL: 'https://metal-slug-maker-default-rtdb.firebaseio.com',
```
Comment (Sam-sta) [inline src/main/server.js:25]: Why you have a comma in here? The lint require it or what? i think it's unnecesary

**Review by Sam-sta — APPROVED** [review]

File: src/main/server.js (hunk 1 above)
Comment (Sam-sta) [reply src/main/server.js:10]: Remember that we use camelCase for var names

Comment (alexeindev) [reply src/main/server.js:10]: Samuel this is a constant, so for standard, it's better to have those in uppercase.

Comment (Sam-sta) [reply src/main/server.js:10]: Ok

**Pull request merged**